h2o-py/h2o/group_by.py

```diff
@@ -50,52 +73,143 @@ def __init__(self, fr, by):
 
 
     def min(self, col=None, na="all"):
+        """
+        Calculate the minimum (per numerical column) of each group of a GroupBy object for column(s) specified by
+        the user.
+
+        :param str/int/list_of_str/list_of_int col: representing column name(s) or column index(indices) that you want
+            to perform aggregation on.  If no col is given to the aggregation, the aggregation would be applied to
+            all columns but the group by columns.
+        :param str na:  one of 'rm', 'ignore' or 'all' (default).
+        :return: aggregation result stored as member _res of the GroupBy object and can be accessed by method
```

Comment:
Actually, this returns the original GroupBy object (`self`), for ease of constructing chained operations.

Internal params such as `_res` should not be documented